src/commcare_cloud/ansible/group_vars/all.yml

```diff
@@ -45,10 +45,11 @@ zookeeper_client_port: 2181
 formplayer_port: 8181
 
 couchdb_use_haproxy: False
+couchdb_use_alb: True
 couchdb_version: 2.3.1
 couchdb2_port: 15984
 couchdb2_local_port: "{{ 15986 if couchdb_version is version('3.0.0', '<') else couchdb2_port }}"
-couchdb2_proxy_port: "{{ 35984 if couchdb_use_haproxy else 25984 }}"
+couchdb2_proxy_port: "{{ 35984 if couchdb_use_haproxy else 80 if couchdb_use_alb else 25984 }}"
```

Comment:
Can you make the ALB listen on port 25984 or just the standard 5984? I think that would be a cleaner change